src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -157,6 +186,40 @@ enum ColumnOfProcessorTypeTable {
     PROC_TYPE_TABLE_SIZE = 4   //!< Size of processor type table
 };
 
+/**
+ * @enum       CpuUseStatus
+ * @brief      Defination of CPU_MAP_USED_FLAG column in CPU mapping table.
+ */
+enum CpuUseStatus {
+    NOT_USED = -1,           //!< CPU is not bound to thread
+    CPU_USED = 1,            //!< CPU is in using
+    PLUGIN_USED_START = 100  //!< Plugin other than CPU needs to use. If more GPUs use CPUs, the CPU_MAP_USED_FLAG is
+                             //!< accumulated from PLUGIN_USED_START. For example: GPU.0:100, GPU.1:101
+};
+
+/**
+ * @brief      Get and reserve available cpu ids
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  core_type core type.
+ * @param[in]  num_cpus number of cpus.
+ * @param[in]  seek_status look for CPU_MAP_USED_FLAG of seek_status in CPU mapping table
+ * @param[in]  reset_status reset CPU_MAP_USED_FLAG with reset_status.
+ * @return     Array of available cpu ids.
+ */
+OPENVINO_RUNTIME_API std::vector<int> reserve_available_cpus(const ColumnOfProcessorTypeTable core_type,
```

Comment:
I suppose we have not agreed about this function on arch forum. Suggest removing

Reply:
Ok, removed now.

Reply:
it's still here

Reply:
Sorry, I misread, this function can not be removed because it is used to reserve cpu cores in CPUStreamsExecutor. I removed this function `reserve_cpu_cores()` which is provided for GPU.

Reply:
The multiple models on CPU case also requires `reserve_available_cpus()`.